docs/sdk-and-tools/indices/delegators.md

```diff
@@ -23,6 +23,18 @@ The `_id` field of this index is composed in this way: `blake2bHash(delegatorAdd
 | contract       | This field holds the bech32 encoded address of the staking provider contract to whom it was delegated to.                            |
 | activeStake    | The activeStake field holds the EGLD amount of the active stake (not undelegated nor unbondable).                                    |
 | activeStakeNum | The activeStake field holds the EGLD amount of the active stake (not undelegated nor unbondable), in a numeric format. Example: 1.5. |
+| unDelegateInfo | The unDelegateInfo contains a list with data about the unDelegated values.                                                           |
+| timestamp      | The timestamp field represents the last moment when was a interation with the delegation contract.                                   |
```

Comment:
The timestamp field represents the last moment of an interaction with the delegation contract.

Reply:
done